src/Uno.UWP/System/DispatcherQueue.cs

```diff
@@ -27,6 +27,8 @@ private DispatcherQueue()
 		public DispatcherQueueTimer CreateTimer()
 			=> new DispatcherQueueTimer();
 
+		internal static DispatcherQueue Main { get; } = new DispatcherQueue(); // TODO: Avoid Main in the future #8978
```

Comment:
Yes for now there is only one, this will change in the future if we start supporting windows on different UI threads, but for now we keep a single UI thread only (actually WinUI itself had recently still issues creating new windows on different thread)